doc/usage/extensions/math.rst

```diff
@@ -140,6 +140,11 @@ are built:
 .. module:: sphinx.ext.mathjax
    :synopsis: Render math using JavaScript via MathJax.
 
+.. warning::
+   Version 4.0 changes the version of MathJax used to version 3. You may need to
+   override ``mathjax_path`` to ``https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS-MML_HTMLorMML``
```

Comment:
Now 2.7.8 is released, so this link is old. I can suggest to use a URL that does not hardcode minor version:

```suggestion
   override ``mathjax_path`` to ``https://cdn.jsdelivr.net/npm/mathjax@2/MathJax.js?config=TeX-AMS-MML_HTMLorMML``
```